plugins/cybereason/icon_cybereason/actions/quarantine_file/action.py

```diff
@@ -16,106 +16,19 @@ def __init__(self):
 
     def run(self, params={}):
         sensor_details = self.connection.api.get_sensor_details(params.get(Input.SENSOR))
-        sensor_guid = sensor_details["guid"]
         malop_id = params.get(Input.MALOP_ID)
         quarantine = params.get(Input.QUARANTINE)
-
         malop_data = self.connection.api.get_malop(malop_id)
+        sensor_guid = sensor_details["guid"]
+        sensor_machine_name = sensor_details["machineName"]
```

Comment:
```suggestion
        sensor_guid = sensor_details.get("guid")
        sensor_machine_name = sensor_details.get("machineName")
```
let's use that way, unless you are pretty sure that we never meet `KeyError` exception